remote/webby/webby_nx_socket_bridge.h

```diff
@@ -0,0 +1,70 @@
+/************************************************************************************
```

Comment:
Note: this bridge is because the Nintendo signatures in the architecture are different than in the generic Nintendo includes. There are additional complications with different architecture include files contain different sets of signatures than the generic or cross platform ones.

Reply:
Not sure this is considered a hack as it seems to me like the right way to do things (given the constraints). You can remove the `BBI-HACK` string unless you're planning to revisit this in the future with a better fitting approach. Since this is debug code, I don't see the need though...

Reply:
resloved and renamed to a different file